ArticleTemplates/assets/js/bootstraps/common.js

```diff
@@ -33,7 +33,7 @@ define([
         comments.init(); // load comments
         cards.init(); // load cards
         loadEmbeds();
-        smoothScroll.init(); // scroll to anchor
+        // smoothScroll.init(); // scroll to anchor
```

Comment:
So this is causing issues on iOS 11? This library is primarily used for scrolling the related cards at the bottom of an article. What happens when it's enabled? Does it cause any issues? I'd rather we enabled it as it'd aloow the ~45% of use on iOS 10 to scroll this area and we can look at the issue on iOS 11 in a separate branch.

Reply:
It throws an `undefined` exception, and so everything after it does not run. Yes I think you're right, I'll amend this by testing the value.